samples/dnn/classification.cpp

```diff
@@ -8,62 +8,65 @@
 
 #include "common.hpp"
 
-std::string param_keys =
-    "{ help  h          | | Print help message. }"
-    "{ @alias           | | An alias name of model to extract preprocessing parameters from models.yml file. }"
-    "{ zoo              | models.yml | An optional path to file with preprocessing parameters }"
-    "{ input i          | | Path to input image or video file. Skip this argument to capture frames from a camera.}"
-    "{ initial_width    | 0 | Preprocess input image by initial resizing to a specific width.}"
-    "{ initial_height   | 0 | Preprocess input image by initial resizing to a specific height.}"
-    "{ std              | 0.0 0.0 0.0 | Preprocess input image by dividing on a standard deviation.}"
-    "{ crop             | false | Preprocess input image by center cropping.}"
-    "{ framework f      | | Optional name of an origin framework of the model. Detect it automatically if it does not set. }"
-    "{ needSoftmax      | false | Use Softmax to post-process the output of the net.}"
-    "{ classes          | | Optional path to a text file with names of classes. }";
-std::string backend_keys = cv::format(
+using namespace cv;
+using namespace std;
+using namespace dnn;
+
+const string about =
+        "Use this script to run a classification model on a camera stream, video, image or in a directory\n\n"
+        "Firstly, download required models using `download_models.py` (if not already done).\n"
+        "To run:\n"
+        "\t ./example_dnn_classification model_name --input=path/to/your/input/image/or/video (don't give --input flag if want to use device camera)\n"
+        "Sample command:\n"
+        "\t ./example_dnn_classification googlenet --input=path/to/image \n"
+        "Model path can also be specified using --model argument";
+
+const string param_keys =
+    "{ help  h         |            | Print help message. }"
+    "{ @alias          |            | An alias name of model to extract preprocessing parameters from models.yml file. }"
+    "{ zoo             | models.yml | An optional path to file with preprocessing parameters }"
+    "{ input i         |            | Path to input image or video file. Skip this argument to capture frames from a camera.}"
+    "{ crop            |   false    | Preprocess input image by center cropping.}";
+
+const string backend_keys = format(
     "{ backend          | 0 | Choose one of computation backends: "
                               "%d: automatically (by default), "
                               "%d: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
                               "%d: OpenCV implementation, "
                               "%d: VKCOM, "
                               "%d: CUDA, "
```

Comment:
can you modify the sample to use text names for backend and target instead of numbers?
e.g.

```
classification -backend openvino -target opencl_fp16 -input lena.jpg
```